WcaOnRails/app/assets/stylesheets/registrations.scss

```diff
@@ -25,3 +26,22 @@ table.registrations-table {
   z-index: 1000; /* show over sticky table headers */
   bottom: calc(5px + #{$footer-height-px});
 }
+
+/* Styling bootstrap table sort */
+.fixed-table-loading {
```

Comment:
It's to hide a message box added by bootstrap-table script, this should be done by an appropriate 'bootstrap-table' stylesheet but including it makes a table no longer looking as 'wca table'. Anyway I will use hideLoading function from API instead of the current solution.